packages/combo-box/src/vaadin-combo-box-scroller.js

```diff
@@ -175,7 +175,7 @@ export class ComboBoxScroller extends PolymerElement {
   }
 
   scrollIntoView(index) {
-    if (!(this.opened && index >= 0)) {
+    if (!this.__virtualizer || !this.opened || index === -1) {
```

Comment:
Why was the check added here? For me, the tests pass without it

Reply:
Hmm, I only added it here to keep the implementation as much closer as possible to the original one, essentially because of this check here:

https://github.com/vaadin/web-components/pull/4543/files#diff-a1439573fecee231c5cc349819720f24f7c001bafe7dfe3048c58ccf140330cdL266-L268

Reply:
Now I'm wondering if these checks originally were necessary especially for `__focusedIndexChanged`

Reply:
Ok, I can probably remove this check from `scrollIntoView` but keep it in `requestContentUpdate` as it is available in public through the combo-box's requestContentUpdate.

Reply:
Wait, we probably don't even need it in the scroller's `requestContentUpdate` because it cannot be invoked anyway when the scroller is not initialized:

https://github.com/vaadin/web-components/blob/64f901e32fc923dcd737c230dc13fb6e4745a48a/packages/combo-box/src/vaadin-combo-box-mixin.js#L368-L378

Reply:
Removed them.

Reply:
I was also trying to find a way to break it with the checks removed but couldn't. Theoretically, a value defined at the end of `ready()` (the `Virtualizer` instance) could be `undefined` in observers that invoke on `super.ready()` phase.

Reply:
Yeah, observers are never invoked before `ready()` because `_updateScroller()` only sets properties after the scroller's initialization.